src/base/utils/misc.h

```diff
@@ -48,6 +48,22 @@ namespace Utils
 {
     namespace Misc
     {
+        // use binary prefix standards from IEC 60027-2
+        // see http://en.wikipedia.org/wiki/Kilobyte
+        enum class SizeUnit
```

Comment:
It may be better to have this enum in the global namespace and to keep here only the functions (i.e., utilities)?